src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -1,148 +1,137 @@
-import * as React from 'react';
-import {StyleSheet, Text, View} from 'react-native';
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
 
-import GlobalButton from '../../../../../components/Button/GlobalButton';
-import {fonts, normalize, normalizeFontSize} from '../../../../../utils/fonts';
-import {COLORS} from '../../../../../utils/theme';
+import GlobalButton from '../../../../../components/Button/GlobalButton'
+import { colors } from '../../../../../utils/colors'
+import { fonts, normalize, normalizeFontSize } from '../../../../../utils/fonts'
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated, which is a good practice to keep the naming consistent. However, it's important to ensure that all instances of `COLORS` have been replaced with `colors` throughout the entire codebase to maintain consistency and avoid potential errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->